executable/executable.go

```diff
@@ -251,6 +260,7 @@ func (e *Executable) Wait() (ExecutableResult, error) {
 		Stdout:   stdout,
 		Stderr:   stderr,
 		ExitCode: e.cmd.ProcessState.ExitCode(),
+		Timeout:  e.ctxWithTimeout.Err() == context.DeadlineExceeded,
 	}, nil
```

Comment:
Error would be a better response here - that way all existing clients will not end up accidentally reading a exit code for an timed-out process (it'll return -1 if so)